src/main/java/zmaster587/advancedRocketry/tile/TileRocketAssemblingMachine.java

```diff
@@ -320,18 +319,20 @@ public void scanRocket(World world, BlockPos pos2, AxisAlignedBB bb) {
 							numBlocks++;
 							
 							//If rocketEngine increaseThrust
+							final float x = xCurr - actualMinX - ((actualMaxX - actualMinX) / 2f);
+							final float z = zCurr - actualMinZ - ((actualMaxZ - actualMinZ) / 2f);
 							if(block instanceof IRocketEngine && (world.getBlockState(belowPos).getBlock().isAir(world.getBlockState(belowPos), world, belowPos) || world.getBlockState(belowPos).getBlock() instanceof BlockLandingPad || world.getBlockState(belowPos).getBlock() == AdvancedRocketryBlocks.blockLaunchpad )) {
 								if (block instanceof BlockNuclearRocketMotor) {
 									nuclearWorkingFluidUseMax += ((IRocketEngine) block).getFuelConsumptionRate(world, xCurr, yCurr, zCurr);
 									thrustNuclearNozzleLimit += ((IRocketEngine)block).getThrust(world, currBlockPos);
-								} else if (block instanceof BlockBipropellantRocketMotor || block instanceof BlockAdvancedBipropellantRocketMotor ) {
+								} else if (block instanceof BlockBipropellantRocketMotor /*|| block instanceof BlockAdvancedBipropellantRocketMotor*/) {
```

Comment:
As long as this code works you don't need to leave the comments in

Reply:
I'll remove the comments.